scanner/src/funTest/assets/scanner-integration-expected-ort-result.yml

```diff
@@ -45,10 +45,38 @@ analyzer:
       scopes:
       - name: "deps"
         dependencies:
+        - id: "Dummy::pkg0:1.0.0"
```

Comment:
Not strictly related to this commit alone, but would it make sense to give the packages names that reflect their properties, instead of using numbers? Like, "pkg0" could be renamed to "pkg-with-empty-vcs".

Reply:
I deliberately went for this setup. This is easy to grasp as the package names align with the directory names and also the files in the repositories. Also, after working on `SpdxDocumentReporterFunTest` I regret the decision that  I've put intention of the test as properties of the package. Finally, also the `pkgs` can be used for testing different things, which are not forsee-able yet. I prefer to keep as-is, since benefit of changing for me is no clear and re-working it is also quite a bit of effort.